src/server/auth/server/api_server.go

```diff
@@ -279,6 +282,7 @@ func (a *apiServer) retrieveOrGeneratePPSToken() {
 }
 
 func (a *apiServer) watchAdmins(fullAdminPrefix string) {
+	b := backoff.NewTestingBackOff()
```

Comment:
Using a testing backoff here seems a little weird to me, we might change it for test related reasons and not realize it can affect the behavior of the rest of the server.

Reply:
This was a total mistake, I meant to use NewExponentialBackoff (fixed!)